src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Waiting.java

```diff
@@ -52,14 +52,8 @@ public void waitForCondition(final String condition, String...params) {
 		if (message == null) {
 			message = String.format("Condition '%s' did not become true in <TIMEOUT>", condition);
 		}
-		waitUntil(timeout, message, new WaitUntilFunction() {
-
-			@Override
-			public boolean isFinished() {
-				return Boolean.TRUE.equals(((JavascriptExecutor) browserManagement.getCurrentWebDriver())
-						.executeScript(condition));
-			}
-		});
+		waitUntil(timeout, message, () -> Boolean.TRUE.equals(((JavascriptExecutor) browserManagement.getCurrentWebDriver())
+				.executeScript(condition)));
```

Comment:
Could you, please, put `waitUntil()` argument to the separate lines to make code a little bit easier to read?

Reply:
Done

Reply:
Done